packages/experimental/Menu/src/MenuTrigger/useMenuTrigger.ts

```diff
@@ -10,6 +14,23 @@ export const useMenuTrigger = (_props: MenuTriggerProps): MenuTriggerState => {
   const open = context.open;
   const openOnHover = context.openOnHover;
   const triggerRef = context.triggerRef;
+  const accessibilityState = context.open ? { expanded: true } : { expanded: false };
+  const onAccessibilityAction = React.useCallback(
+    (e: AccessibilityActionEvent) => {
+      if (Platform.OS === ('win32' as any)) {
+        switch (e.nativeEvent.actionName) {
+          case 'Expand':
+            setOpen(e, true /* isOpen */);
+            break;
+
+          case 'Collapse':
+            setOpen(e, true /* isOpen */);
```

Comment:
Should this be `setOpen(e, false /* isOpen */);` ?

Reply:
Oof yep, thanks for the catch